react/javascript/test/HighlightTest.tsx

```diff
@@ -74,74 +77,18 @@ describe('HighLight', () => {
     ])
   })
 
-  context('when htmlContent is not set', () => {
-    it('escapes HTML characters', () => {
-      const document = renderHighlight(
-        '<span>Given</span> a passed step',
-        'step'
-      )
-      const highlighted = Array.from(
-        document.querySelectorAll('#content .highlight')
-      )
-        .map((span) => span.innerHTML)
-        .join('')
-
-      assert.equal(
-        highlighted,
-        '<span>&lt;span&gt;Given&lt;/span&gt; a passed </span><mark>step</mark>'
-      )
-    })
-
-    it('also highlight the tags', () => {
-      const document = renderHighlight(
-        '<strong>Given</strong> a strong step',
-        'strong'
-      )
-      const highlighted = Array.from(
-        document.querySelectorAll('#content .highlight')
-      )
-        .map((span) => span.innerHTML)
-        .join('')
-
-      assert.equal(
-        highlighted,
-        '<span>&lt;</span><mark>strong</mark><span>&gt;Given&lt;/</span><mark>strong</mark><span>&gt; a </span><mark>strong</mark><span> step</span>'
-      )
-    })
-  })
-
-  context('when htmlContent is set to true', () => {
-    it('keeps the HTML content', () => {
-      const document = renderHighlight(
-        '<em>Given</em> a passed step',
-        'step',
-        true
-      )
-      const highlighted = Array.from(
-        document.querySelectorAll('#content .highlight')
-      )
-        .map((span) => span.innerHTML)
-        .join('')
-
-      assert.equal(highlighted, '<em>Given</em> a passed <mark>step</mark>')
-    })
+  it('puts <mark> around matches in markdown', () => {
+    const document = renderHighlight(
+      '* This is\n* a bullet list',
+      'bullet',
+      true
+    )
+    const highlighted = Array.from(
+      document.querySelectorAll('#content mark')
+    ).map((span) => span.textContent)
 
-    it('does not highlight the tags', () => {
-      const document = renderHighlight(
-        '<strong>Given</strong> a strong step',
-        'strong',
-        true
-      )
-      const highlighted = Array.from(
-        document.querySelectorAll('#content .highlight')
-      )
-        .map((span) => span.innerHTML)
-        .join('')
+    console.log(document.body.innerHTML)
```

Comment:
Assume you didn't mean to leave this console call in?

Reply:
Thanks, gone now.